docs/Manual/03 Schedules.md

```diff
@@ -16,39 +16,40 @@ for i in range(16):
         for k in range(11):
             C[i, j] += A[i, k] * B[k, j]
 ```
-In other words, each of the logical pseudo-code loops in `nest` becomes an actual for-loop in the default schedule.
+In other words, each of the logical pseudo-code loops in `nest` becomes an actual for-loop in the default schedule. 
 
-We can now transform this schedule in various ways. These transformations merely change the order of the loop iterations, and therefore preserve the logic defined in `nest`. Note that transforming `schedule` does not modify `nest`, and in fact, we could generate additional independent schedules by calling `nest.create_schedule()` multiple times.
+We can now transform this schedule in various ways. However, these transformations do not change the underlying logic defined in `nest` and merely change the order of the loop iterations. We can even generate as many independent schedules as we want by calling `nest.create_schedule()`.
 
 ## Iteration spaces: a geometric representation of schedules
-The Accera programming model embraces a geometric interpretation of schedules. Specifically, the schedule is imagined to be a multidimensional discrete hypercube called the *iteration space* of the nest. The elements of the iteration space represent the individual iterations of the loop nest. Initially, the dimensions of the iteration space corresponds to the logical loops defined in `nest`. For example, the default iteration space for the matrix-matrix multiplication nest forms a three dimensional discrete hypercube, whose shape is (16, 10, 11).
+In the Accera programming model, schedules are geometrically interpreted as a multi-dimensional discrete hypercube called the *iteration space* of the nest. The elements of the iteration space represent the individual iterations of the loop nest. Initially, the dimensions of the iteration space correspond to the logical loops defined in `nest`. For example, the default iteration space for the matrix-matrix multiplication nest forms a three-dimensional discrete hypercube, whose shape is (16, 10, 11).
```

Comment:
"In the Accera programming model , a schedule is geometrically interpreted as a multi-dimensional discrete hypercube ..."